src/instructlab/sdg/utils/chunkers.py

```diff
@@ -211,8 +216,17 @@ def chunk_documents(self) -> List:
         if self.document_paths == []:
             return []
 
-        model_artifacts_path = StandardPdfPipeline.download_models_hf()
-        pipeline_options = PdfPipelineOptions(artifacts_path=model_artifacts_path)
+        if not self.docling_model_path.exists():
+            logger.info(
+                f"Docling model path {self.docling_model_path} not found, downloading models..."
+            )
+            os.makedirs(self.docling_model_path, exist_ok=True)
+            self.docling_model_path = StandardPdfPipeline.download_models_hf()
+        else:
+            logger.info("Found the docling models")
+
+        pipeline_options = PdfPipelineOptions(artifacts_path=self.docling_model_path)
+
```

Comment:
Here we can just check if supplied coling_model_path is None or not 
there's probably also no point in ensuring what we supplied exists if it is None so we can just use whatever path is returned to us from `StandardPdfPipeline.download_,models_hf()`

```suggestion
        if self.docling_model_path is None:
            logger.info(
                f"Docling models not found on disk, downloading models..."
            )
            self.docling_model_path = StandardPdfPipeline.download_models_hf()
        else:
            logger.info("Found the docling models")

        pipeline_options = PdfPipelineOptions(artifacts_path=self.docling_model_path)

```